src/main/java/org/fundacionjala/trello/pages/board/MenuMoreBoard.java

```diff
@@ -26,8 +26,8 @@ public boolean isDisplayed() {
     }
 
     public BoardPage closeBoard() {
```

Comment:
I have a question here. Is it necessary to implement a whole class just with one method? Can this method be written in the "Menu board class"?